### DISCUSSION TITLE
Immediate crash on startup, no logs...?

### DISCUSSION THREAD

**Discussion by dunsparce9** [nicotine-plus/nicotine-plus#3184] [Q&A] [answered]
Nicotine 3.3.6 crashes immediately after doubleclick on my laptop. The weird part is it works fine on my home PC, but only crashes on the laptop. The only file it creates is an empty `AppData\Roaming\nicotine\config\config` (0 KB) before crashing and generating a WerFault dump file (attached). 
[dumps.zip](https://github.com/user-attachments/files/17689122/dumps.zip)

Could anyone help me figure this out? :/

Both PCs are Windows 11 24H2 x64.

**Comment by mathiascode** [comment, marked as answer]
Any difference with 3.3.7.dev1? https://nicotine-plus.org/doc/TESTING.html#windows

**Reply by dunsparce9** [reply]
Yep just tried 3.3.7.dev1 and it works fine!! Thank you, didn't know dev builds existed, sorry!
